acceptance/tests/pxp_agent_version.rb

```diff
@@ -1,12 +1,17 @@
 test_name 'C93805 - pxp-agent - Versioning test'
 
-agent1 = agents[0]
+def pxp_version_cmd(platform)
+  if platform.start_with?('windows')
+    # simple hack to ensure OpenSSL is found w/out modifying PATH
+    "cd '/cygdrive/c/Program\\ Files/Puppet\\ Labs/Puppet/pxp-agent/bin' && ../../../pxp-agent/bin/pxp-agent --version"
+  else
+    'cd /opt/puppetlabs/puppet/bin && ./pxp-agent --version'
+  end
+end
 
 step 'cd into pxp-agent bin folder and check the version'
-pxp_agent_folder = ''
-agent1.platform.start_with?('windows') ?
-  pxp_agent_folder = "/cygdrive/c/Program\\ Files/Puppet\\ Labs/Puppet/pxp-agent/bin" :
-  pxp_agent_folder = "/opt/puppetlabs/puppet/bin"
-on(agent1, "cd #{pxp_agent_folder} && ./pxp-agent --version") do |result|
-  assert(/[0-9\.]*/ =~ result.stdout, "Version number should be numbers and periods but was \"#{result.stdout.to_s}\"")
+agents.each do |agent|
```

Comment:
There is no value in repeating the test command on every agent; it was deliberate that only the first agent was tested